### DISCUSSION TITLE
Welcome to quantpi Discussions!

### DISCUSSION THREAD

**Discussion by alienzj** [ohmeta/quantpi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rusher321** [comment]
I think can build a module to specifically support UHGP analysis at the gene/protein level. A higher resolution to quantify maybe is a trend in the future.  Also can consider the gt_pro at the strain level, but the quantify pipeline in this tool needs to further develop.

**Reply by alienzj** [reply]
Yes, I agree with you. A higher resolution is needed. 
The most of the classifier/profiler focus on the genome level, and it is hard to get the details of gene/protein on the context of genome quantification. As far as I know, perhaps Humann3 is the only widely used tool for metagenomic functional analysis at genefamilies/pathway level. But if you want to assemble some genomes, and want to know the contribution of unknown species represented by these genomes to the function of the whole community, it is more troublesome in practice.

For gene/protein quantification based on the `UHGP` database, maybe we can refer [geneshot](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02355-6) or [gene-level architectures](https://www.nature.com/articles/s41467-021-23029-8), or [MetaQUBIC](https://academic.oup.com/bioinformatics/article/35/21/4474/5497255), or any BGI-meta publication to study it. It is more direct to study directly at the gene level and to statistically analyze a large number of genes of interest, which can be compared to RNA-seq gene quantitative analysis and differential analysis.

For `gt_pro`, I haven't figured out this method yet, but I plan to put it in `callpi`. Of course, it may be more convenient to use it in `quantpi`, because it belongs to the topic of profiling. If you are interested, welcome to develop together. As so-and-so said, SNP/SNV analysis is the next frontier of microbiome research. More and more bacterial genome variation, driving evolution, adaptive evolution and colonization are discovered in microbiome research. Great !

**Comment by alienzj** [comment]
@rusher321 metawibele: https://www.nature.com/articles/s41586-022-04648-7

**Reply by alienzj** [reply]
metawibele workflow: 
![image](https://user-images.githubusercontent.com/8847420/170604927-849fe046-78b5-4d14-aff4-4f8cb3b6b6a5.png)

**Reply by rusher321** [reply]
A quick view, I sure that's we discussed above of the quantification in gene level of the gene family,  and indicate us how can we do after the prediction or annotation of new genes or proteins assembled; 
After a detailed reading,  let's have a further discussion.
